lib/librato/rails/subscribers/job.rb

```diff
@@ -0,0 +1,15 @@
+module Librato
+  module Rails
+    module Subscribers
+      hooks = %w{enqueue_at enqueue perform_start perform}
+
+      hooks.each do |hook|
+        ActiveSupport::Notifications.subscribe "#{hook}.active_job" do |*args|
+          collector.group 'rails.job' do |c|
+            c.increment hook
```

Comment:
Hmm, if we do add that I'd favor making it a beta thing, wonder how useful a cross-job timing would be, probably want to be able to break those out by job name or something - we could prob use job name as source if it is available.